src/dmd/dsymbol.d

```diff
@@ -476,6 +476,9 @@ extern (C++) class Dsymbol : ASTNode
      * `toParent3()` returns a logically enclosing scope symbol this is a member of.
      * It skips over TemplateMixin's.
      *
+     * `toParent4()` similar to `toParent2()` but always follows the template declaration scope
```

Comment:
Don't worry about this in this PR, but we really should name these better at some point.

Reply:
for sure it's confusing enough